inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -884,19 +910,8 @@ Accepting equal validity from self-reported and card-reported vaccination status
 ```{r vaccination_coverage_age_group}
 # return weighted counts and weighted proportions 
 survey_design %>% 
-  tabulate_survey(vaccination_status_simple, age_group, deff = TRUE, digits = 2, wide = FALSE) %>%
-  
-  # spread values to wide format to have appropriate groups 
-    # collect variables of interest (n, deff and ci) (call the clasifier column "variable")
-  gather(key = "variable", value = "value", c(n, deff, ci)) %>% 
-    # create temp column which combine the classifier column with vacc_status column seperated by underscore
-  unite(col = "temp", vaccination_status_simple, variable, sep = "_") %>%
-    # spread wide based on newly create combo column above 
-  spread(temp, value) %>% 
-  # change order of columns 
-  select(age_group, Vaccinated_n, Vaccinated_ci, Vaccinated_deff, 
-         Unvaccinated_n, Unvaccinated_ci, Unvaccinated_deff) %>% 
-  
+  tab_survey(vaccination_status_simple, strata = age_group, deff = TRUE, digits = 2,
+             drop = "Missing", transpose = "value") %>%
   rename("Age group (years)" = age_group) %>% 
   augment_redundant(" (n)"         = "_n") %>%    # wrap all "n" vars in braces (note space before n).
```

Comment:
just flagging for myself - for some reason comment here is duplicated a couple lines down randomly